include/mpd/playlist.h

```diff
@@ -59,6 +59,25 @@ struct mpd_connection;
  */
 struct mpd_playlist;
 
+/**
+ * MPDs queue save modes.
+ *
+ * @since libmpdclient 2.21, MPD 0.24.
+ */
+enum mpd_queue_save_mode {
+	/** Create new playlist */
+	MPD_QUEUE_SAVE_MODE_CREATE = 0,
+
+	/** Replace existing playlist */
+	MPD_QUEUE_SAVE_MODE_REPLACE,
+
+	/** Append to existing playlist */
+	MPD_QUEUE_SAVE_MODE_APPEND,
+
+	/** Unknown state */
+	MPD_QUEUE_SAVE_MODE_UNKNOWN
```

Comment:
What does that mean?

Reply:
Incorrect description of this enum, fixed.